src/components/FindWallet/WalletTable/index.tsx

```diff
@@ -0,0 +1,573 @@
+import { FaDiscord, FaGlobe, FaTwitter } from "react-icons/fa"
+import {
+  Box,
+  chakra,
+  Flex,
+  FlexProps,
+  forwardRef,
+  Icon,
+  Img,
+  keyframes,
+  SimpleGrid,
+  SimpleGridProps,
+  Table,
+  TableProps,
+  Td,
+  Text,
+  Th,
+  Tr,
+} from "@chakra-ui/react"
+
+// Components
+import Link, { IProps as LinkProps } from "../../Link"
+import { WalletMoreInfo } from "./WalletMoreInfo"
+
+// Icons
+import {
+  GreenCheckProductGlyphIcon,
+  WarningProductGlyphIcon,
+} from "../../icons/staking"
+
+// Utils
+import { useWalletTable } from "./useWalletTable"
+import { trackCustomEvent } from "../../../utils/matomo"
+import { getImage } from "../../../utils/image"
+import { WalletData } from "../../../data/wallets/wallet-data"
+import { ChildOnlyProp } from "../../../types"
+
+const Container = (props: TableProps) => (
+  <Table
+    w="full"
+    sx={{
+      th: {
+        fontWeight: "normal",
+        p: {
+          fontSize: "0.9rem",
+        },
+      },
+    }}
+    {...props}
+  />
+)
+
+const WalletContainer = (props: ChildOnlyProp) => (
+  <Container
+    borderBottom="1px"
+    borderColor="lightBorder"
+    _hover={{ bg: "boxShadow", transition: "0.5s all" }}
+    {...props}
+  />
+)
+
+const Grid = forwardRef<SimpleGridProps, "tr">((props, ref) => (
+  <SimpleGrid
+    as={Tr}
+    ref={ref}
+    templateColumns={{ base: "60% auto 0% 0% 5%", md: "40% auto auto auto 5%" }}
+    w="full"
+    columnGap={2}
+    alignItems="center"
+    {...props}
+  />
+))
+
+const WalletContentHeader = (props: ChildOnlyProp) => (
+  <Grid
+    bg="background"
+    borderBottom="1px"
+    borderColor="primary"
+    templateColumns={{
+      base: "auto",
+      sm: "60% auto 0% 0% 5%",
+      md: "40% auto auto auto 5%",
+    }}
+    rowGap={{ base: 4, sm: 0 }}
+    p={2}
+    position="sticky"
+    top={0}
+    zIndex="docked"
+    sx={{
+      th: {
+        p: 0,
+        borderBottom: "none",
+        color: "currentColor",
+        fontSize: "0.9rem",
+        lineHeight: "revert",
+        letterSpacing: "revert",
+        textAlign: "revert",
+        textTransform: "revert",
+        "&:nth-of-type(2)": {
+          display: { base: "flex", sm: "revert" },
+          alignItems: "center",
+          gap: 4,
+        },
+        "&:nth-of-type(3), &:nth-of-type(4)": {
+          hideBelow: "sm",
+        },
+      },
+    }}
+    {...props}
+  />
+)
+
+const Wallet = forwardRef<ChildOnlyProp, "tr">((props, ref) => (
+  <Grid
+    ref={ref}
+    cursor="pointer"
+    py="25px"
+    px={{ base: 4, lg: 1 }}
+    sx={{
+      p: {
+        m: 0,
+      },
+      td: {
+        padding: 0,
+        borderBottom: "none",
+        height: "full",
+      },
+      "td:nth-of-type(3), td:nth-of-type(4)": {
+        hideBelow: "sm",
+      },
+    }}
+    {...props}
+  />
+))
+
+const ChakraSelect = chakra((props: { className?: string }) => (
+  <Select {...props} />
+))
+const StyledSelect = (props) => (
+  <ChakraSelect
+    sx={{
```

Comment:
![image](https://github.com/ethereum/ethereum-org-website/assets/468158/7bd69417-4797-476d-8087-35e5c8d2cd78)

```suggestion
  <ChakraSelect
    w="full"
    sx={{
```